packages/denylist/src/index.js

```diff
@@ -0,0 +1,53 @@
+/* eslint-env serviceworker */
+
+import { Router } from 'itty-router'
+
+import { denylistGet } from './denylist.js'
+import { versionGet } from './version.js'
+
+import { addCorsHeaders, withCorsHeaders } from './cors.js'
+import { errorHandler } from './error-handler.js'
+import { envAll } from './env.js'
+
+const router = Router()
+
+// https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent
+/** @typedef {ExecutionContext} Ctx */
+
+router
+  .all('*', envAll)
+  .get('/version', withCorsHeaders(versionGet))
+  .get('/', withCorsHeaders(denylistGet))
```

Comment:
Per feedback above, let's do:

```suggestion
  .get('/:cid', withCorsHeaders(denylistGet))
```

Reply:
```suggestion
```

So this line should now go away

Reply:
i kept this only to give API users a more meaningful error message if they request `/` without providing a cid.  but i guess from an HTTP perspective, if they request `/`, that's technically a different resource.  so 404 would be acceptable.